models/ios/ospf/ios_ospf.yaml

```diff
@@ -0,0 +1,741 @@
+---
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+
+DOCUMENTATION: |
+module: ios_ospf
+version_added: 2.10
+short_description: Manages and configures Open Shortest Path First (OSPF) and it's attributes.
+description: This module configures and manages the Open Shortest Path First (OSPF) on IOS platforms.
+author: Sumit Jaiswal (@justjais)
+notes:
+  - Tested against Cisco IOSv Version 15.2 on VIRL
+  - This module works with connection C(network_cli).
+    See L(IOS Platform Options,../network/user_guide/platform_ios.html).
+options:
+  config:
+    description: A dictionary of OSPF options.
+    type: list
+    elements: dict
+    suboptions:
+      ospf_type:
+        description:
+        - OSPF version to be used
+        - OSPFv2 supports only IPV4 configuration
+        - OSPFv3 supports both IPV4 and IPV6 configuration
+        choices:
+          - ospf
+          - ospfv3
+      id:
```

Comment:
Shouldn't this be under the `instances` key?

Reply:
yea, I haven't updated the respective model yet with updated changes